prime-router/src/main/resources/metadata/hl7_mapping/OML_O21/OML_O21-base.yml

```diff
@@ -34,6 +34,14 @@ elements:
     resourceIndex: patientIndex
     schema: classpath:/metadata/hl7_mapping/OML_O21/base/patient/patient-base.yml
 
+  - name: insurance
+    resource: 'Bundle.entry.resource.ofType(Coverage)'
+    #    todo: update logic when Coverage.subscriber is mapped (IN1-16/IN1-17)
```

Comment:
Please include the ticket number of where the issue will be addressed in TODOs so we can ensure it is tracked.

Reply:
Added in a link to https://github.com/CDCgov/prime-reportstream/issues/15500